torch/_torch_docs.py

```diff
@@ -9704,6 +9706,10 @@ def merge_dicts(*dicts):
 :attr:`input` tensor, so changing the content of one would change the content
 of the other.
 
+.. seealso::
+
+        :func:`torch.t`.
```

Comment:
add a half sentence here, too

Reply:
Added, will update the screenshots once the preview gets generated.